susemanager-utils/testing/automation/backend-unittest-pgsql.sh

```diff
@@ -1,7 +1,25 @@
 #! /bin/sh
```

Comment:
```suggestion
#! /bin/sh

SCRIPT=$(basename ${0})
```
So when we show the help, we don't show the full path that includes the PWD